views/mixins/overviewFields.jade

```diff
@@ -0,0 +1,10 @@
+mixin overviewDetails(fields)
+	each fieldset, key in fields
+		section.group
+			h1=key
+			div.body
+				each field in fieldset
+					div.field
+						p
+							span.col-sm-2.label=field.label
```

Comment:
These 2 lines show be wrapped in a `row` class. See https://getbootstrap.com/css/#grid for more info on the structure of the grid. Perhaps the class `overview-details` should also have a class named `container`